### DISCUSSION TITLE
part-ii/2019-48/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#198] [Question comments]
Part II 2019 Differential Geometry: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2019-48/

**Comment by ArthurConmy** [comment]
I believe that the pretty much the same counterexample works for i), ii) and iii):
Take the set of points
$$(t, \exp(-1/t^2), 0)$$
for $t > 0$ and $(t, 0, \exp(-1/t^2))$ for $t < 0$ as well as $(0,0,0)$ as an arc-length parametrised curve. Then (manually checking) this has $k(s)=0$ at $s=0$ and is smooth (all derivatives vanish). Clearly $\tau=0$ except at $0$. Moreover, using the set of points $(t, \sqrt{2}/2 \exp(-1/t^2), \sqrt{2}/2 \exp(-1/t^2))$ will produce a second curve not related by a smooth motion with the same properties.
